aten/src/ATen/mps/MPSStream.mm

```diff
@@ -10,6 +10,9 @@ @interface MPSGraphExecutionDescriptor ()
 
 namespace at::mps {
 
+// threshold to perform adaptive commit if the accumulated size
+// of resources encoded on the command buffer exceeds that.
+static const size_t kCmdBufAdaptiveCommitThreshold = MB(64);
```

Comment:
In future we would like to extend the heuristic to add operations (flops) as well as bytes (read/written) as heuristic.